bpf/process/generic_calls.h

```diff
@@ -1,4 +1,5 @@
-/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright Authors of Cilium */
 #include "bpf_tracing.h"
```

Comment:
you changed this in previous comment, please change it directly to final value and make it separate commit, it will ease the review

Reply:
Resolved after re-split commits